bazaarvoice/emodb
# PD-3742 split postman collection

We have a very long 100k+ rows file with all the tests that have been implemented at the moment.
We have decided to split one file with code to several files (postman collections) in order to simplify code review process as well as use it for test run parallelization.  

## How to Test and Verify

Check out this PR
Start emodb on local
Open postman
Import into postman any of next files as a collection:
"emodb_tests_databus.postman_collection.json"
"emodb_tests_databus_get_claimcount.postman_collection.json" 
"emodb_tests_databus_get_peek.postman_collection.json"
"emodb_tests_databus_get_poll.postman_collection.json"
"emodb_tests_databus_get_size.postman_collection.json"
"emodb_tests_databus_get_subscription.postman_collection.json"
"emodb_tests_databus_unsubscribe.postman_collection.json"
"emodb_tests_sor.postman_collection.json"
"emodb_tests_sor_facade.postman_collection.json"
"emodb_tests_sor_imports_arbitary_size.postman_collection.json"
"emodb_tests_sor_table.postman_collection.json"
"emodb_tests_sor_tableName.postman_collection.json"
Import "sor-1-_table.postman_environment.json" as environment variables
Execute tests in postman

Profit
Expected result: All tests are successfully executed on local environment


## Risk
Low
There are no changes to application code base

### Level 

`Low`, `Medium`, or `High`. Give an indication of what you think is the level of change introduced by this PR. `High` means a massive change to a core functionality. 
`Low` means a really minor change that shouldn't have any regression effect. 

### Required Testing

All existing postman tests should be executed without failures.

### Risk Summary

There are no risks to application, only tests are involved and it is about splitting existing tests

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
